bytecomp/bytegen.ml

```diff
@@ -345,11 +345,11 @@ let comp_primitive p args =
   | Pfloatcomp Cgt -> Kccall("caml_gt_float", 2)
   | Pfloatcomp Cle -> Kccall("caml_le_float", 2)
   | Pfloatcomp Cge -> Kccall("caml_ge_float", 2)
-  | Pstringlength -> Kccall("caml_ml_string_length", 1)
-  | Pstringrefs -> Kccall("caml_string_get", 2)
-  | Pstringsets -> Kccall("caml_string_set", 3)
-  | Pstringrefu -> Kgetstringchar
-  | Pstringsetu -> Ksetstringchar
+  | Pstringlength | Pbyteslength -> Kccall("caml_ml_string_length", 1)
+  | Pstringrefs | Pbytesrefs -> Kccall("caml_string_get", 2)
+  | Pstringsets | Pbytessets -> Kccall("caml_string_set", 3)
+  | Pstringrefu | Pbytesrefu -> Kgetstringchar
+  | Pstringsetu | Pbytessetu -> Ksetstringchar
```

Comment:
These should be `bytes`, not `byte`, but I'm not sure renaming the C functions is worth the trouble.